docs/docs/parachains/picasso/token-transparency.md

```diff
@@ -0,0 +1,138 @@
+the Picasso ecosystem, network security, application development and other forms of utility. 
+This distribution involves the Picasso Network treasury receiving transaction fees. 
+This is governed by the Picasso General Council, which is intended to later be governed by PICA token holders. 
+75% of spent network fees will automatically flow into the treasury pallet, 
+with the remaining 25% of spent fees intended to be distributed to collators. 
+Here, the token holders have the ability to identify and submit for approval improvement proposals relating to the Picasso network.
+Improvement proposals may include but are not limited to implementing additional security audits, 
+grants for development on the Picasso network, core protocol updates or additions, 
+new integrations of other ecosystems and protocols into Picasso and the broader Composable ecosystem, 
+educational initiatives, further developer and/or user documentation, 
+additional technological exploration and enhancement, etc. 
+Token Holders are encouraged to discuss initial ideas and recommendations for improvement proposals via conversation 
+on the dedicated Picasso Discord and submit formal proposals through Picasso’s governance process.
+
+For the purposes of advancing these above objectives,
+
+* Composable may re-balance Composable-controlled accounts when warranted. 
+  This could involve creating new addresses and/or removing old addresses. 
+  For the sake of transparency, any changes in account addresses will be reflected below on this page.
+* Composable may liquidate PICA on occasion, in the event that expenses cannot be paid in this token. 
+  For instance, Composable may liquidate PICA to accumulate KSM for maintaining the Picasso parachain slot on Kusama.
+
+## Genesis Addresses
+
+The following is a list of non-circulating and circulating genesis addresses of PICA tokens. 
+Upon token launch, the genesis pools would be the multisig addresses below. 
+We have published this list for the community to monitor for purposes of transparency. 
+The other non-multisig addresses are labeled as protocol controlled addresses. 
+
+### Non-Circulating Genesis Addresses
+
+The following multisigs are controlled by the governance council and are activated only by governance. 
+These multisigs are considered non-circulating, 
+and can only be considered “circulating” if voted upon by protocol governance to become circulating.
+
+Treasury Multisig: 5wCLknMQkBGmuTQxsxYAyevpa37mMCkFLaqaYNcowsSnR7CJ 
+
+Team Tokens Multisig: 5wyQr1huDp4mKAKA95ihXRqWu8hLkqjiNib8GaKFzuEMKEk3
+
+Ecosystem Incentives Multisig : 5ytUzQtNj7vibyr79cqUENuDLNUnTsgDAJMDefhBywRTNrDU
+
+Liquidity Programs Multisig : 5ytrBrTBma4kKfmEXGoKGMVjfyTnKgpvXaiwwCbXCbdZQK25
+
+Treasury Pallet for accruing fees: 5wCLknMQkBGmuTQxsxYAyevpa37mMCkFLaqaYNcowsSnR7CJ
+
+### Protocol Controlled Addresses
+
+#### Crowdloan Pallet Funds
+
+Are located at the crowdloan pallet: 
```

Comment:
when we have this address, please provide 

and also link to the csv of the rewards as well

Reply:
@0xBrainjar2 Where can I get this or who can I ask for this address?